app/templates/components/footer-nav.hbs

```diff
@@ -5,58 +5,117 @@
         <div class="row flex-items-xs-center">
           <div class="col-12 col-sm-6 col-md-5 mt-3" itemscope itemtype="http://schema.org/Organization" itemref="social-button-facebook social-button-instagram social-button-twitter">
             <div id="organisation-alpha">
+
               <h5 class="footer-header" itemprop="name"> {{t 'C.S.V. Alpha'}} </h5>
               <meta itemprop="url" content="https://csvalpha.nl">
               <meta itemprop="logo" content="/images/alphalogo.svg">
+
               <table class="table table-sm">
                 <tbody>
                   <tr>
-                    <td class="footer-table-cell"><i class="fa fa-envelope" aria-hidden="true"></i></td>
+                    <td class="footer-table-cell">{{fa-icon 'envelope'}}</td>
```

Comment:
Don't we need all the aria-hidden?

Reply:
Those are included when using `{{fa-icon}}`. We needed those on every FA icon anyway

Reply:
Ah great! LGTM!